x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -305,6 +306,14 @@ export const getSavedObjectTypes = (
             },
           ],
         },
+        '5': {
+          changes: [
+            {
+              type: 'mappings_addition',
+              addedMappings: {},
```

Comment:
```suggestion
              addedMappings: {
                dynamic: false,
              },
```
You should also specify the change in mappings here. It's unfortunate that we have this duplication of the mappings definitions, but we didn't see a good way around it.

https://github.com/elastic/kibana/blob/skip-transform-on-mappings-change/docs/developer/architecture/core/saved-objects-service.asciidoc?plain=1#L324-L325

Reply:
I tried this but I got a type error `Type 'boolean' is not assignable to type 'SavedObjectsFieldMapping'` Should we ignore that error?

Reply:
ah I see, you're right, looks like a limitation in the modelVersions API in that you can't actually specify a change that sets the top-level dynamic option.